docs/en-US/component/upload.md

```diff
@@ -99,7 +99,7 @@ upload/manual
 | `headers`                         | request headers.                                                                                                                                                                      | `Headers \| Record<string, any>`                                                              | —        | No       |
 | `method`                          | set upload request method.                                                                                                                                                            | `string`                                                                                      | `'post'` | No       |
 | `multiple`                        | whether uploading multiple files is permitted.                                                                                                                                        | `boolean`                                                                                     | `false`  | No       |
-| `data`                            | additions options of request.                                                                                                                                                         | `Record<string, any>`                                                                         | —        | No       |
+| `data`                            | additions options of request.                                                                                                                                                         | `Awaitable<Record<string, any>> \| ((rawFile: UploadRawFile) => Awaitable<Record<string, any>>)`                                                                         | —        | No       |
```

Comment:
```suggestion
| `data`                            | additions options of request. support `Awaitable` data and  `Function` since  v3.3.13                                                                                                                                     | `Record<string, any> \| Awaitable<Record<string, any>> \| ((rawFile: UploadRawFile) => Awaitable<Record<string, any>>)`                                                                         | —        | No       |
```